bin/start-helloworld

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+demo_dir="$( cd "$(dirname "$0")/.." ; pwd -P )"
+
+export CONJUR_AUTHN_API_KEY=$(\
+    docker-compose --project-directory "$demo_dir" \
+    run -T conjur-cli host rotate_api_key --host helloworld \
```

Comment:
This could be `--host $DEMO_HELLOWORLD_ID` or something, to make it configurable.

Reply:
👍